Makefile

```diff
@@ -244,7 +244,8 @@ extension-test: extension-test-build
 	$(if $(filter Windows_NT,$(OS)),\
 		set "E2E_TEST_FILES_DIRECTORY=extension" &&,\
 		E2E_TEST_FILES_DIRECTORY=extension) \
-    ctest --test-dir build/$(call get-build-path,RelWithDebInfo)/test/runner --output-on-failure -j ${TEST_JOBS} --exclude-regex "${EXTENSION_TEST_EXCLUDE_FILTER}"
+    ctest --test-dir build/$(call get-build-path,RelWithDebInfo)/test/runner --output-on-failure -j ${TEST_JOBS} --exclude-regex "${EXTENSION_TEST_EXCLUDE_FILTER}" && \
+    ctest --test-dir build/$(call get-build-path,RelWithDebInfo)/extension --output-on-failure -j ${TEST_JOBS} --exclude-regex "${EXTENSION_TEST_EXCLUDE_FILTER}" && \
```

Comment:
Can you try:
```suggestion
    ctest --test-dir build/$(call get-build-path,RelWithDebInfo)/{test/runner,extension} --output-on-failure -j ${TEST_JOBS} --exclude-regex "${EXTENSION_TEST_EXCLUDE_FILTER}" && \
```

Reply:
This makes the command much simpler, Thanks!

Reply:
@sdht0 It doesn't work on linux, only works for mac
https://github.com/kuzudb/kuzu/actions/runs/16211680005/job/45774020874?pr=5731